Fin update

jaredthomas68 opened the pull request:
This PR updates the financial parameters for both HOPP and GreenHEART to be location specific for each reference design location. This PR should be reviewed and merged after #2 and #3

kbrunik commented on reference-systems/01-minnesota-steel/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,24 +18,24 @@ system_costs:
   federal_tax_rate: 21.0 
-  state_tax_rate: 8.84
-  property_tax_rate: 1.0
+  state_tax_rate: 9.8
```
Do we want to include sources for these values that are location specific?

kbrunik commented on reference-systems/02-texas-ammonia/greenHEART/input-files/plant/greenheart_config_onshore_tx.yaml:
```diff
@@ -17,21 +17,21 @@ project_parameters:
-  debt_equity_ratio: 1.72
-  property_tax: 0.01  # percent of CAPEX # combined with property insurance then between H2A and H2FAST defaults
+  debt_equity_ratio: 2.62
+  property_tax: 0.0147  # percent of CAPEX # combined with property insurance then between H2A and H2FAST defaults
```
Was this not available for Texas?

kbrunik commented on reference-systems/02-texas-ammonia/greenHEART/input-files/plant/greenheart_config_onshore_tx.yaml:
```diff
@@ -17,21 +17,21 @@ project_parameters:
+  property_tax: 0.0147  # percent of CAPEX # combined with property insurance then between H2A and H2FAST defaults
   property_insurance: 0.005 # percent of CAPEX # combined with property tax then between H2A and H2FAST defaults
-  total_income_tax_rate: 0.2574 # 0.257 tax rate in 2022 atb baseline workbook
+  total_income_tax_rate: 0.21 # 0.257 tax rate in 2022 atb baseline workbook
```
Was this not available for Texas?

Is this tax rate from the 2024 ATB workbook?

kbrunik commented on reference-systems/02-texas-ammonia/greenHEART/input-files/plant/greenheart_config_onshore_tx.yaml:
```diff
@@ -17,21 +17,21 @@ project_parameters:
+  total_income_tax_rate: 0.21 # 0.257 tax rate in 2022 atb baseline workbook
   capital_gains_tax_rate: 0.15 # H2FAST default
-  sales_tax_rate: 0.0 
+  sales_tax_rate: 0.082 
```
What's this from?

kbrunik commented on reference-systems/04-new-york-bight-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,20 +18,20 @@ system_costs:
-  admin_expense_percent_of_sales: 0.0
-  capital_gains_tax_rate: 15.0
-  debt_type: "Revolving debt"
+  sales_tax_rate_state: 8.20 # average combined state and local sales tax in NJ
```
Is NJ the same as Texas?

kbrunik commented on reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,20 +18,20 @@ system_costs:
 financial_parameters:
   inflation_rate: 0.0 # nominal basis
-  real_discount_rate: 0.11 # nominal return based on 2022 ATB basline workbook
+  real_discount_rate: 0.063 # nominal return based on 2024 ATB basline workbook
```
Update line to include`with 0% inflation, real and nominal rates are equal` and which tab on the workbook it came from

kbrunik commented on reference-systems/05-california-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,24 +18,24 @@ system_costs:
 financial_parameters:
   inflation_rate: 0.0 # nominal basis
-  real_discount_rate: 0.11 # nominal return based on 2022 ATB basline workbook
+  real_discount_rate: 0.063 # nominal return based on 2024 ATB basline workbook
```
Update line to `real_discount_rate: 0.063 # nominal return based on 2024 ATB baseline workbook - with 0% inflation, real and nominal rates are equal`

kbrunik commented on reference-systems/01-minnesota-steel/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,24 +18,24 @@ system_costs:
-  state_tax_rate: 8.84
-  property_tax_rate: 1.0
+  state_tax_rate: 9.8
+  property_tax_rate: 1.18
```
Where is this from?

kbrunik commented on reference-systems/03-gulf-of-mexico-hydrogen/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,20 +18,20 @@ system_costs:
   federal_tax_rate: 21.0 
-  state_tax_rate: 8.84
-  property_tax_rate: 1.0
+  state_tax_rate: 0.0
```
Add where `state_tax_rate` and `property_tax_rate` are from

kbrunik commented on reference-systems/05-california-hydrogen/greenHEART/input-files/plant/greenheart_config_offshore_ca.yaml:
```diff
@@ -15,15 +15,15 @@ project_parameters:
-  debt_equity_ratio: 1.72
-  property_tax: 0.01  # percent of CAPEX # combined with property insurance then between H2A and H2FAST defaults
+  debt_equity_ratio: 2.62
+  property_tax: 0.0071  # percent of CAPEX # combined with property insurance then between H2A and H2FAST defaults
```
This doesn't seem like it's from H2A and H2FAST?

kbrunik reviewed (changes requested):
Okay I figured I'd leave some overall comments rather than commenting on each little thing:

1.  Add which tab from 2024 ATB the values came from i.e. "Offshore Wind", "Land-Based Wind"
2. I see 2022 ATB still on comments, lets update those to 2024 ATB values or fix the comment
3. Generally going over the comments again, it seems like some should be site specific but say H2A and H2FAST defaults.
4. Do we want to include URLs to where some of the site specific numbers are coming from?

jaredthomas68 commented:
> Okay I figured I'd leave some overall comments rather than commenting on each little thing:
> 
> 1. Add which tab from 2024 ATB the values came from i.e. "Offshore Wind", "Land-Based Wind"
> 2. I see 2022 ATB still on comments, lets update those to 2024 ATB values or fix the comment
> 3. Generally going over the comments again, it seems like some should be site specific but say H2A and H2FAST defaults.
> 4. Do we want to include URLs to where some of the site specific numbers are coming from?

I believe I have addressed all of your comments. Thank you for your feedback.

kbrunik commented on reference-systems/01-minnesota-steel/greenHEART/input-files/plant/greenheart_config_onshore_mn.yaml:
```diff
@@ -16,22 +16,22 @@ project_parameters:
   administrative_expense_percent_of_sales: 0.00 # percent of sales H2FAST default
   depreciation_method: "MACRS" # can be "MACRS" or "Straight line" - MACRS may be better and can reduce LCOH by more than $1/kg and is spec'd in the IRS MACRS schedule https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
-  depreciation_period: 7 # years - as specified by the IRS MACRS schedule https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
+  depreciation_period: 5 # years - as specified by the IRS MACRS schedule for wind and solar https://www.irs.gov/publications/p946#en_US_2020_publink1000107507
```
And battery?

kbrunik commented on reference-systems/01-minnesota-steel/greenHEART/input-files/plant/default_fin_config.yaml:
```diff
@@ -18,24 +18,24 @@ system_costs:
-  insurance_rate: 0.5
-  debt_percent: 63.23
-  term_int_rate: 5.0
+  real_discount_rate: 9.0 # nominal return based on 2024 ATB baseline workbook for land-based wind - with 0% inflation, real and nominal rates are equal
```
Presumably you are using the values for cost year 2030 and the moderate case? Could be good to add that as well

kbrunik reviewed (approved):
Just a few comment that would be good to address before merging. Thanks for all of the added detail!
